scripts/react_native_pods.rb

```diff
@@ -149,7 +149,7 @@ def use_react_native_codegen!(spec, options={})
   return if ENV['DISABLE_CODEGEN'] == '1'
 
   # The path to react-native (e.g. react_native_path)
-  prefix = options[:path] ||= File.join(__dir__, "..")
+  prefix = options[:path] ||= "../../node_modules/react-native"
```

Comment:
the path here seems wrong to me, should just be `../../`, no? This path would be relative to `FBReactNativeSpec.podspec` also in general i think it's kinda dangerous to infer this path anyway, it would be safer for the `.podspec` file to always just specify this path rather as it has the full context of where it is exactly relative to `react_native_path`